deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -649,9 +649,7 @@ bool IsolateShim::RunSingleStepOfReverseMoveLoop(v8::Isolate* isolate,
                                                  int64_t* nextEventTime) {
     int64_t snapEventTime = -1;
     int64_t snapEventEndTime = -1;
-    int64_t origNETime = *nextEventTime;
-    // TODO(mrkmarron) JsTTDMoveMode is no longer 64 bit wide, fix use of
```

Comment:
Is this comment still relevant? I don't see any other changes to JsTTDMoveMode in this review.

Reply:
This comment was stale so I wanted to clean it up.